### PR TITLE
[Protostar] Add offline template as example

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Improvement.
#### Summary of Changes

Add a default offline template for protostar to serve as an example.

image
#### Testing Instructions
1. Use latest staging
2. Install patch
3. Play with offline options in global configuration and check the result in the frontend
4. Test login and login errors
5. Test also customizing protostar template
6. Test also in mobile view
7. Test also in a rtl language
#### Observations

If any frontend developer can review it, i'm open to any suggestions and/or improvements.

### CODE REVIEW

**Comment by brianteeman** [comment]
I have an error on the page
screen shot 2016-04-12 at 14 11 09

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9877">issues.joomla.org/joomla-cms/9877</a>.</sub>

**Comment by andrepereiradasilva** [comment]
and without this patch how does your offline pae appears?

File: templates/protostar/offline.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+{
+	$doc->addStyleSheetVersion('templates/' . $this->template . '/css/user.css');
+}
+
+// Load optional RTL Bootstrap CSS
+JHtml::_('bootstrap.loadCss', false, $this->direction);
+
+// Logo file or site title param
+$sitename = $app->get('sitename');
+if ($this->params->get('logoFile'))
```
Comment (zero-24) [inline templates/protostar/offline.php]: Can we have a clean line here bevor the if?

**Comment by brianteeman** [comment]
Screenshot without the patch - i dont remenber seeing the error before - must be getting old
5qjv

**Comment by andrepereiradasilva** [comment]
so your problem is something else, not to do with this PR

**Comment by brianteeman** [comment]
doing a clean install now of latest staging

On 12 April 2016 at 22:20, andrepereiradasilva <notifications@github.com>
wrote:

> so your problem is something else, not to do with this PR
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/9877#issuecomment-209107725>
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by andrepereiradasilva** [comment]
i just tried yet again on latest staging, didn't get any issue

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on b8265f352f5e0c265aa27bbcad88627a9b3c6087<br /><br />Must have been something wrong i my test install - all good now on latest staging<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9877">issues.joomla.org/joomla-cms/9877</a>.</sub>

**Comment by ghazal** [comment]
I have tested this item :white_check_mark: successfully on b8265f352f5e0c265aa27bbcad88627a9b3c6087<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9877">issues.joomla.org/joomla-cms/9877</a>.</sub>

**Comment by andrepereiradasilva** [comment]
sorry guys, forgot to add offline.php fiel to templatedetails.xml.

No real change as you can check.
https://github.com/joomla/joomla-cms/pull/9877/commits/e75d950b3cd1af3f702b00a8bab300abb4f1abdd

So your tests are still success.

**Comment by brianteeman** [comment]
Looks good to me - setting RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9877">issues.joomla.org/joomla-cms/9877</a>.</sub>

**Pull request merged**